src/utils/loremIpsum.ts

```diff
@@ -0,0 +1,7 @@
+export const loremIpsum = `Lorem Ipsum is simply dummy text of the printing and typesetting industry.
```

Comment:
Could also use faker:

```ts
import faker from 'faker'

const dummyText = faker.lorem.sentence(4)
```

Reply:
not good for storybook snapshots

Reply:
oh kak yes you're right. I'm using them all over the place. will research if it's possible to 'freeze' faker for snapshots

Reply:
cool